EMOBON00092/manual_example_ro-crate-metadata.json

```diff
@@ -1,9 +1,14 @@
-{ "@context": "https://w3id.org/ro/crate/1.1/context", 
+{ "@context": {
+    "@base":"https://github.com/emo-bon/",

```

Comment:
should become base of the ro-crate being built as it is going to be publised via data.emobon.embrc.eu

so rather:
```json
"@base":"https://data.emobon.embrc.eu/mgf-s3-prototype/EMOBON00092/",
```